doc/classes/AudioServer.xml

```diff
@@ -35,6 +35,26 @@
 				Generates an [AudioBusLayout] using the available buses and effects.
 			</description>
 		</method>
+		<method name="get_absolute_time" qualifiers="const" experimental="">
+			<return type="float" />
+			<description>
+				Returns the absolute time in seconds of the [AudioServer] timeline based on the number of audio frames mixed. Used to schedule sounds to be played in the future with higher accuracy.
```

Comment:
"schedule" implies "in the future", I assume.
I think the possessive "s" is appropriate here?
```suggestion
				Returns the absolute time in seconds of the [AudioServer]'s timeline, based on the number of audio frames mixed. Used to schedule sounds to be played with higher accuracy.
```
Part of me is inclined to mention AudioStreamPlayer's new method here, but in practice we'd have to reference it from all 3 classes, which is a bit strange.

Reply:
Perhaps we can just reference `AudioStreamPlayer` and leave it at that? As a user of Godot, it's not too hard to figure out that the methods in all 3 types of players have similar methods.

Reply:
Yeah, that's fine.